src/Config.cpp

```diff
@@ -202,6 +202,7 @@ void Config::Load()
         AddConfigVar("CheckUpdateInterval", config.checkUpdateInterval, lineView, foundProperty);
 
         AddConfigVar("TimeSpace", config.timeSpace, lineView, foundProperty);
+        AddConfigVar("DateTimeLocale", config.dateTimeLocale, lineView, foundProperty);
```

Comment:
Can you put that below to ```DateTimeStyle``` to the other string properties?

Reply:
Okay, have moved it up.